__device-tests__/helpers/utils.js

```diff
@@ -159,11 +159,28 @@ const typeString = async ( driver: wd.PromiseChainWebdriver, element: wd.Promise
 
 const typeStringIos = async ( driver: wd.PromiseChainWebdriver, element: wd.PromiseChainWebdriver.Element, str: string, clear: boolean ) => {
 	if ( clear ) {
-		await element.clear();
+		//await element.clear(); This was not working correctly on iOS so need a custom implementation
+		await clearTextBox( driver, element );
 	}
 	await element.type( str );
 };
 
+const clearTextBox = async ( driver: wd.PromiseChainWebdriver, element: wd.PromiseChainWebdriver.Element ) => {
+	await element.click();
+	let originalText = await element.text();
+	let text = originalText;
+	// We are double tapping on the text field and pressing backspace until all content is removed.
+	do {
+		originalText = await element.text();
+		const action = new wd.TouchAction( driver );
+		action.tap( { el: element, count: 2 } );
```

Comment:
Why isn't a 1 time double tap not enough out of the while loop at the beginning?

Reply:
The method is called tap, so I assumed that I need to 2 there do get a double tap...

Reply:
Oh I meant something different. I wanted to ask if it's working like this?

```
do
- double tap text field
- delete 1 character
- double tap text field again
- delete 1 more character
- double tap text field again
- delete 1 more character
while....
```

Reply:
I'm doing a double tap, because it selects full words, so that way the clear should be quicker.